docs/data-ai/ai/alert.md

```diff
@@ -7,3 +7,231 @@ type: "docs"
 no_list: true
 description: "TODO"
 ---
+
+On this page, you'll learn how to use triggers to send alerts in the form of email notifications when certain detections or classifications are made.
+
+You will build a system that can monitor camera feeds and detect situations that require review.
+In other words, this system performs anomaly detection.
+Whenever the system detects an anomaly, it will send an email notification.
+
+First, you'll set up and test the computer vision functionality.
+Next, you'll set up data capture and sync to record images with the anomaly and upload them to the cloud.
+Finally, you'll configure a trigger to send email notifications when the anomaly is detected.
+
+## Requirements
+
+### Required hardware
+
+- A camera such as a standard USB webcam.
+  You can also test the anomaly detection system using the webcam built into your laptop.
+- A computer capable of running [`viam-server`](/installation/viam-server-setup/).
+  You can use a personal computer running macOS or Linux, or a single-board computer (SBC) running 64-bit Linux.
+
+### Optional hardware
+
+If you want to set up your camera far away from your personal computer, you can use a webcam plugged into a single-board computer, powered by batteries.
+You could mount your machine in a stationary location like on a pole, or you could mount it on a rover.
+This tutorial covers the software side; you can get creative with the hardware.
+
+Note that your machine must be connected to the internet for data sync and email notifications to work.
+
+## Set up your monitor
```

Comment:
like I said yesterday, this feels weird - let's iterate

Reply:
Ah sorry this edit slipped my mind, will make it